src/Editor/CodelensProvider.ts

```diff
@@ -19,6 +19,7 @@ import toolsAttr from './json/tools_attr.json';
 
 export class CodelensProvider implements vscode.CodeLensProvider {
   showTool: Array<string> = [];
+  notShowAttr: Array<string> = [];
```

Comment:
Could you add some comments, in the source code, about relation of `showTool` and `notShowAttr` ?
I could think of `if there is showTool, why would we use notShowAttr if notShowAttr = !showTool`...
But I think there would be different meanings so some comment would help understand this.

Reply:
```
// Save the name of the tool to view the description of the attribute.
showTool: Array<string> = []; 
// Save properties that will not be shown.
notShowAttr: Array<string> = [];
```

I would like to add as follows.

Reply:
Ah...  how about `hideAttr`  ?

Reply:
Thank you for your good opinion!
I will change the variable name and add comments.

Reply:
Comments looks good :)